src/status_im/hardwallet/core.cljs

```diff
@@ -101,6 +428,14 @@
            (= 6 numbers-entered))
       (proceed-to-pin-confirmation)
 
+      (and (= enter-step :current)
+           (= 6 numbers-entered))
```

Comment:
can we create some constants fro magic numbers?

Reply:
done